modules/channel-web/src/views/lite/store/composer.ts

```diff
@@ -40,6 +40,11 @@ class ComposerStore {
     return this.rootStore.config?.composerPlaceholder
   }
 
+  @computed
+  get composerMaxTextLength(): number {
+    return this.rootStore.botInfo?.maxMessageLength
```

Comment:
Since you typed this as a number, should return a number. `|| 0` or some other default constant